app/views/partials/homepage.html

```diff
@@ -106,8 +106,8 @@ <h2 class="text-center">Prestations locales</h2>
             </div>
           </div>
           <div class="col-md-12 showDetailsButtonBlock">
-            <button class="btn" type="button" ng-show="!showDetails && partenaireLocal.prestations.length > 2" ng-click="showDetails = ! showDetails">
-              Voir les {{ partenaireLocal.prestations.length - 2 }} autres aides calculées
+            <button class="btn" type="button" ng-show="!showDetails && nbPrestations(partenaireLocal) > 2" ng-click="showDetails = ! showDetails">
```

Comment:
`partenaireLocal.prestation` is now an `Object`, and I don't think `Object.keys(partenaireLocal.prestations).length` works inside an angular expression.